src/components/tabs/tabs.js

```diff
@@ -8,39 +8,33 @@ const tabs = [
   'Forging',
 ];
 
-const getTabs = (isDelegate, labels) => (labels.filter((item, idx) => idx !== 2 || isDelegate));
+const getTabs = isDelegate => tabs.filter(t => t !== 'Forging' || isDelegate);
 
-const getIndex = history => (
+const getIndex = history =>
   tabs.map(t => t.toLowerCase())
-    .indexOf(history.location.pathname.replace('/main/', '')));
+    .indexOf(history.location.pathname.split('/')[2]);
 
-const isCurrent = (history, index) => history.location.pathname.replace('/main/', '') === tabs[index].toLowerCase();
+const isCurrent = (history, index) =>
+  history.location.pathname.indexOf(tabs[index].toLowerCase()) === 6; // after: /main/
 
 const navigate = (history, index) => {
   if (!isCurrent(history, index)) {
-    history.push(`${tabs[index].toLowerCase()}`);
+    history.push(`/main/${tabs[index].toLowerCase()}`);
   }
 };
 
-const Tabs = (props) => {
-  const tabsLabel = [
-    props.t('Transactions'),
-    props.t('Voting'),
-    props.t('Forging'),
-  ];
-  return (
-    <ToolboxTabs index={getIndex(props.history)}
-      theme={styles}
-      onChange={navigate.bind(this, props.history)}
-      className={`${styles.tabs} main-tabs`}>
-      {getTabs(props.isDelegate, tabsLabel).map((tab, index) =>
-        <Tab
-          key={index}
-          label={tab}
-          className={styles.tab}
-          disabled={isCurrent(props.history, index)} />)}
-    </ToolboxTabs>
-  );
-};
+const Tabs = ({ history, isDelegate, t }) => (
+  <ToolboxTabs index={getIndex(history)}
+    theme={styles}
+    onChange={navigate.bind(null, history)}
+    className={`${styles.tabs} main-tabs`}>
+    {getTabs(isDelegate).map((tab, index) =>
+      <Tab
+        key={index}
+        label={t(tab)}
```

Comment:
Calling `t(tab)` doesn't allow i18n-scanner to find tab names for the source*.json file.

Reply:
Done